data/logos/help-using-a-module.txt

```diff
@@ -0,0 +1,15 @@
+ ______________________________________
```

Comment:
Is this 8m late or 4m early? :trollface:

Reply:
Both 😄

Just making sure it's available in the kali/parrot/nightly major releases ahead of time, as I was a bit too late last year :+1: